src/main/java/seedu/weme/model/meme/Description.java

```diff
@@ -5,11 +5,11 @@
 
 /**
  * Represents a Meme's weme in the meme book.
- * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
+ * Guarantees: immutable; is valid as declared in {@link #isValidDescription(String)}
  */
-public class Address {
+public class Description {
 
-    public static final String MESSAGE_CONSTRAINTS = "Addresses can take any values, and it should not be blank";
+    public static final String MESSAGE_CONSTRAINTS = "Descriptions can take any values, and it should not be blank";
```

Comment:
We should allow descriptions to be empty strings if we are not using `Optional`.